marathon_acme/clients.py

```diff
@@ -226,41 +226,35 @@ def raise_for_not_ok_status(response):
     return response
 
 
-def sse_content_with_protocol(response, callbacks):
+def sse_content_with_protocol(response, handler):
```

Comment:
If this isn't public API, should it perhaps be named `_sse_content_with_protocol` instead?